hr_employee_social_media/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
@feketemihai Thanks for the comment. I agree, it could be more usefull in your way.